pkg/vault/test/foundry/E2eSwap.t.sol

```diff
@@ -348,7 +384,7 @@ contract E2eSwapTest is BaseVaultTest {
         uint256 poolSwapFeePercentage;
     }
 
-    function _testDoUndoExactInBase(uint256 exactAmountIn, DoUndoLocals memory testLocals) private {
+    function _testDoUndoExactInBase(uint256 exactAmountIn, DoUndoLocals memory testLocals) internal {
         if (testLocals.shouldTestDecimals) {
             decimalsTokenA = bound(testLocals.newDecimalsTokenA, 6, 18);
```

Comment:
The tests pass with a minimum of 5 decimals and could pass with 4 decimals with a minor adjustment. I left the tests with 5 decimals, just in case, to say that we're being cautious, hahaha

Reply:
Actually, with 5 decimals I need to reduce the error tolerance for the tests in the weighted pools, so I returned to 6 decimals.